spec/factories/users.rb

```diff
@@ -69,7 +69,7 @@
       after(:build) { |user| user.created_at = 3.weeks.ago }
     end
 
-    trait :ignore_after_callback do
+    trait :ignore_mailchimp_subscribe_callback do
```

Comment:
@maestromac I thought it's better to give this trait a name that better reflects what it actually does.